### ISSUE TITLE
Visual indicator for beginners

### ISSUE THREAD

**Issue by MiiMimlam** [budlabs/typiskt#16]
Feature request.

Visual indicator that displays the fingers that should be used. Indicator can be toggled.
 
```
  '
||||     ||||
|  |/   \|  |
`           `
```

**Comment by budRich** [comment]
I have thought about this and it would be really nice. The problem is that keyboard layouts vary in really unpredictable ways, but i will investigate how it could be done. 

### brainstorm begin:

A raw way of doing it is having "indication.theme" files that would simply look something like this:
```
# numbers are colors
thumb=1
ring=3
pinky=2
mod=4

char[q]=pinky-L
char[Q]=pinky-L-RS
char[w]=ring-L
char[W]=ring-L-RS
```

and then making a script that generate these theme files from some commonly used keyboard layout program. I don't know, its a lot of work, and still it will never be 100% and i guess the feature would have to be disabled by default, so not the wrong layout is loaded for a "beginner" the first time they start the program...

There is also beside hardware and software keyboard layouts, different fingerpositioning schemes...

**Comment by budRich** [comment]
maybe the best would be to create a standalone GUI program so that one can see an image of the actual keyboard and the fingerpositions when selecting..

**Comment by MiiMimlam** [comment]
> char[Q]=pinky-L-RS
```
'           '
||||     ||||
|  |/   \|  |
`           `
```
above, the ascii art image -> pinky's and on the first line presented indicator symbol `'` could be color coded.

**Comment by budRich** [comment]
Yeah, the aesthetics can be done in many different ways. One is to change the color of the actual character, but that would be a little bit too much lolcat. I was also thinking that it might be nice with 4 + 1 + 4 squares (two halfblocks `██`) where the middle one indicates thumb.
